ipatests/test_integration/test_installation.py

```diff
@@ -498,6 +500,42 @@ def test_selinux_avcs(self):
             # Use expected failure until all SELinux violations are fixed
             pytest.xfail("{} AVCs found".format(len(avcs)))
 
+    def test_file_permissions(self):
+        args = [
+            "rpm", "-V",
+            "python3-ipaclient",
+            "python3-ipalib",
+            "python3-ipaserver"
+        ]
+
+        if osinfo.id == 'fedora':
+            args.extend([
+                "freeipa-client",
+                "freeipa-client-common",
+                "freeipa-common",
+                "freeipa-server",
+                "freeipa-server-common",
+                "freeipa-server-dns",
+                "freeipa-server-trust-ad"
+            ])
+        else:
+            args.extend([
+                "ipa-client",
+                "ipa-client-common",
+                "ipa-common",
+                "ipa-server",
+                "ipa-server-common",
+                "ipa-server-dns"
+            ])
+
+        result = self.master.run_command(args, raiseonerr=False)
+        if (result.returncode != 0):
+            mode_warnings = re.findall(
+                r"^.M.......  [cdglr ]+ (?P<filename>.*)$",
```

Comment:
I don't understand the purpose of this check. Is it ignoring some modifications?